src/Symfony/Component/Messenger/Tests/DependencyInjection/MessengerPassTest.php

```diff
@@ -494,12 +494,14 @@ public function testRegistersMiddlewareFromServices()
         $container = $this->getContainerBuilder($fooBusId = 'messenger.bus.foo');
         $container->register('middleware_with_factory', UselessMiddleware::class)->addArgument('some_default')->setAbstract(true);
         $container->register('middleware_with_factory_using_default', UselessMiddleware::class)->addArgument('some_default')->setAbstract(true);
+        $container->register('middleware_with_factory_using_a_lot_of_defaults', UselessMiddleware::class)->setArguments(array('one', 'two', 'three'))->setAbstract(true);
```

Comment:
`middleware_with_factory_replacing_defaults_by_index` ? 😅 ("Using a lot of defaults" doesn't really convey the intent here)